client/components/Reviews/Reviews.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Reviews from './Reviews';
+import exampleGetReviews from './exampleGetReviews';
+
+const exampleReviews = exampleGetReviews.results;
+
+it('should render a reviews list', () => {
+  render(<Reviews reviews={exampleReviews} />);
+
+  const reviews = screen.getByTestId('reviews');
+  const reviewsList = screen.getByTestId('reviews-list');
+  const reviewsInList = screen.getAllByTestId('review');
+
+  expect(reviews).toBeInTheDocument();
+  expect(reviewsList).toBeInTheDocument();
+  expect(reviewsInList).toHaveLength(5);
```

Comment:
When I run the test, this  `expect(reviewsInList).toHaveLength(5);` gave me an error (maybe because it only renders two reviews at first even though there are 5 reviews in total.